package.json

```diff
@@ -17,8 +17,6 @@
     "concurrently": "^8.2.2"
   },
   "dependencies": {
-    "grommet": "^2.39.0",
-    "grommet-icons": "^4.12.1",
-    "styled-components": "^6.1.11"
+    "eslint": "^9.8.0"
```

Comment:
in this top level package.json we can't have any other dependencies except "concurrently": "^8.2.2" as "dev dependency", 

if you believe, you still need "eslint" dependency - put it under "devDependencies" in either "client" or "server" - exactly where it's needed.